### ISSUE TITLE
kwin config with configured hotkeys?

### ISSUE THREAD

**Issue by Hi-Angel** [faho/kwin-tiling#67]
I know, [because of kwin bug](https://bugs.kde.org/show_bug.cgi?id=340225) scripts can't yet register keys on their own.

Then it'd be great to have a working `.kksrc` config *(i.e. an export of kwin hotkeys)*. Because keys are pretty much alike in most tilling WMs *(except for the <kbd>Mod</kbd> key)*. Or even better, a kksrc-style commented list of keys to configure, because it's

1. Hard to configure so many keys with GUI
2. Even so, I've no slightest idea, how to configure layout switch hotkey *(and no, <kbd>Meta</kbd>+<kbd>PgUp</kbd> doesn't work for me)*.
3. I can't even get a scroll through the windows working: whilst <kbd>Ctrl</kbd>+<kbd>k</kbd> working as expected, its the opposite, <kbd>Ctrl</kbd>+<kbd>j</kbd> switches only between the last 2 windows.
4. I've no idea how to configure floating mode hotkey

Basically, I just can't start using KDE because of hotkeys.

**Comment by faho** [comment]
Sorry, but that wouldn't work because we need to register the callbacks, which won't exist until the script has done so. And it can't do so because that method currently does not work.

The "floating mode hotkey" is one such thing - kwin has no idea of a "floating mode", this script needs to register it. And it needs to do so at runtime.
